torchdynamo/config.py

```diff
@@ -18,11 +18,14 @@
     HAS_REFS_PRIMS = False
 
 
+INFO_CODE = 15
```

Comment:
Unless we want this to be configurable, we should hide this elsewhere in TorchDynamo, maybe utils.py?

Reply:
Or even it's own module, we've said this a few times for other things, so we could create a logging_utils module or something.